source/services/odnoklassniki.js

```diff
@@ -20,7 +20,7 @@ const odnoklassniki = {
     popupHeight: 400,
 };
 
-set(window, 'ODKL.updateCount', (index, counter) => {
+set(global, 'ODKL.updateCount', (index, counter) => {
```

Comment:
> window is currently used to share the data between different instances of Likely

Oh, got it, didn't know. I agree that's in that case it's more complex issue. Better not to touch it :)